inscriptions/ISic003373.xml

```diff
@@ -184,13 +184,15 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM"> <!-- Greek? -->
```

Comment:
Is this a case where, in line with the Elymian files, you should put a certainty element on the textlang element further up in the file, and suggest it could be either Greek or Sikel for now? Can you add a visible note in the commentary (inside a `<p>` element) that this stone appears to be unpublished and inventory number not yet identified.

Reply:
I would also note that this is a piece which I'd be very happy if you were to publish it formally at some  point, assuming we can find out something more about its provenance, etc. (I've flagged it at least once to Angela Maria as one with a big question mark next to it).

Reply:
happy to do it!